src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/SelectAccount.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@model SFA.DAS.LevyTransferMatching.Web.Models.Opportunities.SelectAccountViewModel
```

Comment:
The header needs changing so that the blue upper part is shown, while the nav bar is hidden.

Reply:
This has been done.